docs/getting-started/first-abn.md

```diff
@@ -31,25 +31,23 @@ A sample application using the Iter8 SDK is provided. Deploy both the frontend a
 
     === "node"
         ```shell
-        kubectl create deployment frontend --image=iter8/abn-sample-frontend-node:0.15.0
+        kubectl create deployment frontend --image=iter8/abn-sample-frontend-node:0.17.3
```

Comment:
Should these be 18.3?

Reply:
No, this is in docs repo so 17 still makes sense.